engine/src/exec-interface2.cpp

```diff
@@ -2480,8 +2480,18 @@ bool MCInterfaceStringCouldBeStack(MCStringRef p_string)
          MCStringBeginsWithCString(p_string, (const char_t *)"REVO",
                                   kMCCompareExact)))
         return true;
-    
-    // Check if it could be a script-only stack
+
+    //TODO: process BOM at the point of import into LiveCode (from URL)
```

Comment:
I'm not sure this is necessary given the way this PR works - it wouldn't really add anything, and is actually quite tricky given that (at present) we basically treat the StringRef as Data in several of these functions (which is correct because of the way Data<->String conversions work, just a little bit murky).